test/namenode/NameNodeTest.cc

```diff
@@ -0,0 +1,110 @@
+#include "zkwrapper.h"
+#include "zk_nn_client.h"
+
+#include <gtest/gtest.h>
+
+#include <cstring>
+#include <easylogging++.h>
+
+INITIALIZE_EASYLOGGINGPP
+
+namespace {
+
+	class NamenodeTest : public ::testing::Test {
+
+	protected:
+		virtual void SetUp() {
+			system("sudo ~/zookeeper/bin/zkServer.sh stop");
```

Comment:
Please move these system calls into main(). Set up stuff should be called before RUN_ALL_TESTS(); tear down stuff should be called afterwards. This will allow for much faster tests. But, it might require you to change some of the paths so that they're unique across tests.